cmake.deps/cmake/BuildTreesitterParsers.cmake

```
@@ -23,3 +23,4 @@ endfunction()
 BuildTSParser(c ${TREESITTER_C_URL} ${TREESITTER_C_SHA256} TreesitterParserCMakeLists.txt)
 BuildTSParser(lua ${TREESITTER_LUA_URL} ${TREESITTER_LUA_SHA256} TreesitterParserCMakeLists.txt)
 BuildTSParser(vim ${TREESITTER_VIM_URL} ${TREESITTER_VIM_SHA256} TreesitterParserCMakeLists.txt)
+BuildTSParser(vimdoc ${TREESITTER_VIMDOC_URL} ${TREESITTER_VIMDOC_SHA256} TreesitterParserCMakeLists.txt)
```

```suggestion
BuildTSParser(help ${TREESITTER_VIMDOC_URL} ${TREESITTER_VIMDOC_SHA256} TreesitterParserCMakeLists.txt)
```
will generate `help.so` instead.

---

Thanks! `./build/lib/nvim/parser/help.so` exists now, but still `vim.treesitter.require_language('help')` and `vim.treesitter.get_parser(buf, 'help')` fails.

---

`parser_spec` might be picking up your _installed_ `c.so` from your system Nvim?

---

`:echo nvim_get_runtime_file('**/parser/c.so',v:true)`

---

Indeed, `parser_spec.lua` fails if I just `make` but do not `make install`.

---

Interesting, so maybe we need to set 'runtimepath' for the tests. Looks like it's ok on CI though, so not a blocker for now.